azure-pipelines.yml

```diff
@@ -85,7 +85,7 @@ stages:
 
         envs:
           - macos: py37
-          - windows: py38
+          - windows: py310
```

Comment:
Let us see if this is a mistake.

Reply:
Isn't this likely to make it even slower?

Reply:
Nope.